tests/unit/inference/test_inference.py

```diff
@@ -139,8 +141,25 @@ def invalid_model_task_config(model_w_task, dtype, enable_cuda_graph):
 @pytest.fixture
 def query(model_w_task):
     model, task = model_w_task
+    angle_bracket_mask_models = [
+        "roberta",
+        "camembert",
+        "esm",
+        "ibert",
+        "luke",
+        "mpnet",
+        "yoso",
+        "mpnet"
+    ]
+
+    def check_angle_bracket_mask(model):
+        for name in angle_bracket_mask_models:
+            if name in model:
+                return True
+        return False
+
     if task == "fill-mask":
-        if "roberta" in model:
+        if check_angle_bracket_mask(model):
```

Comment:
```suggestion
        if any(map(lambda x: x in model, angle_bracket_mask_models)):
```
We could avoid having `check_angle_bracket_mask` with this one-liner